src/mmrelay/db_pool.py

```
@@ -0,0 +1,312 @@
+                            break
+                        time.sleep(0.1)
+
+                    if connection is None:
+                        raise sqlite3.OperationalError(
+                            "Connection pool exhausted and timeout reached"
+                        )
+
+            yield connection
+
+        except Exception as e:
+            logger.error(f"Error in connection pool: {e}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)